src/main/java/tr/com/orioninc/laborant/controller/RestLabController.java

```diff
@@ -0,0 +1,32 @@
+package tr.com.orioninc.laborant.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import tr.com.orioninc.laborant.service.LabService;
+
+@RestController
+@RequestMapping("/v1")
+@AllArgsConstructor
+public class RestLabController {
+
+    private LabService labService;
+
+    @GetMapping("/labs/status")
+    public ResponseEntity<String> getAllLabsStatus() {
+        return ResponseEntity.ok(labService.getAllLabsStatus());
+    }
+    @GetMapping("/labs/runCommand/{labName}/{command}")
+    public ResponseEntity<String> runCommand(@PathVariable("labName") String labName, @RequestParam String command) {
+        try {
+            return ResponseEntity.ok(labService.runCommandOnSelectedLab(labName, command));
+        } catch (Exception e) {
+            return ResponseEntity.badRequest().body(e.getMessage());
```

Comment:
* dont return exception to client, return more meaningful user friendly messages
* log the exception
* Don't catch class Exception, operate through the actual trowed exceptions

Reply:
What happened to try/catch here?

Reply:
Method in service class already checking the exception and customer already getting a message when couldn't connect to the lab etc.